packages/icons/readme.md

```diff
@@ -0,0 +1,43 @@
+# Cypress Icons
+
+The latest versions of the icons.
+The public API will always reference these files.
+
+`./dist` is not checked into source control.
+
+## API
+
+```js
+icons = require("@cypress/icons")
```

Comment:
```suggestion
const icons = require("@cypress/icons")
```

Reply:
Nit, but nice to have valid code in the examples so you can copy paste.

Reply:
agreed